Update win_package.ps1

richardsonky opened the pull request:
Update Test-Path to use -LiteralPath instead of -Path to fix issue where powershell will not detect path with special characters such as '=' and '[]'.

##### SUMMARY
Fix for #62521

##### ISSUE TYPE
- Bugfix Pull Request


##### COMPONENT NAME
win_package

##### ADDITIONAL INFORMATION
If the path variable is contains characters such as '=' or '[]' powershell can't verify the path using Test-Path.  The powershell command needs to be Test-Path -LiteralPath for it verify it correctly.

richardsonky committed:
Update win_package.ps1

Update Test-Path to use -LiteralPath instead of -Path to fix issue where powershell will not detect path with special characters such as '=' and '[]'.

jborean93 commented:
Thanks for the PR, there's still a few other locations where `-Path` is being used when `-LiteralPath` is an option.

* https://github.com/ansible/ansible/blob/cd4da7df5176418831fb500534fb8b3a87d70ccb/lib/ansible/modules/windows/win_package.ps1#L140
* https://github.com/ansible/ansible/blob/cd4da7df5176418831fb500534fb8b3a87d70ccb/lib/ansible/modules/windows/win_package.ps1#L243
* https://github.com/ansible/ansible/blob/cd4da7df5176418831fb500534fb8b3a87d70ccb/lib/ansible/modules/windows/win_package.ps1#L233
* https://github.com/ansible/ansible/blob/cd4da7df5176418831fb500534fb8b3a87d70ccb/lib/ansible/modules/windows/win_package.ps1#L235
* https://github.com/ansible/ansible/blob/cd4da7df5176418831fb500534fb8b3a87d70ccb/lib/ansible/modules/windows/win_package.ps1#L253
* https://github.com/ansible/ansible/blob/cd4da7df5176418831fb500534fb8b3a87d70ccb/lib/ansible/modules/windows/win_package.ps1#L257
* https://github.com/ansible/ansible/blob/cd4da7df5176418831fb500534fb8b3a87d70ccb/lib/ansible/modules/windows/win_package.ps1#L366
* https://github.com/ansible/ansible/blob/cd4da7df5176418831fb500534fb8b3a87d70ccb/lib/ansible/modules/windows/win_package.ps1#L391
* https://github.com/ansible/ansible/blob/cd4da7df5176418831fb500534fb8b3a87d70ccb/lib/ansible/modules/windows/win_package.ps1#L454
* https://github.com/ansible/ansible/blob/cd4da7df5176418831fb500534fb8b3a87d70ccb/lib/ansible/modules/windows/win_package.ps1#L479

richardsonky committed:
Update win_package.ps1

modified other instances of -Path and changed to -LiteralPath.  All except line L243 since it is a different function.
